### PR TITLE
Add wizard support

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

File: cms_form/tests/utils.py [hunk 1]
```diff
@@ -32,6 +33,24 @@ def fake_request(form_data=None, query_string=None,
     return o_req
 
 
+def fake_session(env, **kw):
+    db = get_db_name()
+    env = api.Environment(env.cr, env.uid, {})
+    session = http.root.session_store.new()
+    session.db = db
+    session.uid = env.uid
+    session.login = 'admin'
```
Comment (grindtildeath) [inline cms_form/tests/utils.py]: Are we sure `env.uid` above matches with admin ? What about using this with `sudo()` in a test ?

File: cms_form/models/cms_form_wizard.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    def wiz_current_step(self):
+        return self.wiz_storage_get().get('current') or 1
+
+    def wiz_next_step(self):
+        return self.wiz_storage_get().get('next')
+
+    def wiz_prev_step(self):
+        return self.wiz_storage_get().get('prev')
+
+    def form_next_url(self, main_object=None):
```
Comment (grindtildeath) [inline cms_form/models/cms_form_wizard.py:119]: Here it seems next does not have the same meaning as above at line 112 (wiz_next_step). Not sure if you can use something different like form_following_url ?

File: cms_form/templates/form.xml [hunk 3]
```diff
@@ -235,4 +235,49 @@ License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 </template>
 
 
+<template id="wizard_form_wrapper" name="CMS wizard form wrapper"
+          inherit_id="cms_form.form_wrapper" primary="True">
+
+  <t t-if="form.form_title" position="before">
+    <t t-if="form.wiz_show_progress_bar">
```
Comment (grindtildeath) [inline cms_form/templates/form.xml:242]: Nitpicking : Here we get to `t` elements with `t-if` right after, but one is a selector and the second an addition. Using `xpath`element for first would make this more obvious.

File: cms_form/tests/fake_models.py [hunk 4]
```diff
@@ -90,3 +89,58 @@ def _form_validate_a_float(self, value, **request_values):
...
+
+class FakeWizStep3Partner(models.AbstractModel):
+
+    _name = 'fake.wiz.step3.partner'
+    _inherit = 'fake.wiz'
+    _form_model = 'res.partner'
+    _form_model_fields = ('name', )
+
+
+WIZ_KLASSES = [
```
Comment (grindtildeath) [inline cms_form/tests/fake_models.py:143]: Is this german ? Any reason no to use 'classes' as in english ?

**Review by grindtildeath — COMMENTED** [review]
Just a few questions...

File: cms_form/tests/fake_models.py (hunk 4 above)
Comment (simahawk) [reply cms_form/tests/fake_models.py:143]: convention to not mess up w/ built-in names and when you search for `class` in your code you don't get false results too.

File: cms_form/models/cms_form_wizard.py (hunk 2 above)
Comment (simahawk) [reply cms_form/models/cms_form_wizard.py:119]: `form_next_url` comes from cms form base implementation. It stays for "where do you go next after you submit the form".
In wizard case it simply has a similarity w/ `next` action but is not related actually.

File: cms_form/templates/form.xml (hunk 3 above)
Comment (simahawk) [reply cms_form/templates/form.xml:242]: I avoid xpaths when possible, I think this is more readable ;P

**Review by tschanzt — APPROVED** [review]

**Review by grindtildeath — APPROVED** [review]
LGTM :+1:

**Pull request merged**